examples/eigenproblems/eigenproblems_ex3/eigenproblems_ex3.C

```diff
@@ -110,9 +110,9 @@ int main (int argc, char ** argv)
   libmesh_example_requires(false, "--disable-singleprecision");
 #endif
 
-#ifdef LIBMESH_USE_COMPLEX_NUMBERS
-  // SLEPc currently gives us an "inner product not well defined" with
-  // Number==complex
+#ifdef LIBMESH_USE_COMPLEX_NUMBERS && SLEPC_VERSION_LESS_THAN(3,6;2)
```

Comment:
That semicolon between 6 and 2 should be a comma!

Reply:
Also isn't the `&&` logic wrong here? You want to trigger the `libmesh_example_requires` statement if complex numbers are enabled *or* SLEPc is too old, yes?

Reply:
oh, you are absolutely right; strange that it compiled...

No, the logic should be right; it was stated that it doesn't work with complex numbers; but in my case (I checked with 3.6.2) it works fine. I'll check with 3.8.3 as well, but I assume that it works for newer versions so it should be skipped only for older versions and if complex numbers are used.

Reply:
OK